graylog2-server/src/main/java/org/graylog2/alerts/types/FieldContentValueAlertCondition.java

```diff
@@ -88,10 +88,9 @@ public ConfigurationRequest getRequestedConfiguration() {
     public static class Descriptor extends AlertCondition.Descriptor {
         public Descriptor() {
             super(
-                "Field Content Value Alert Condition",
+                "Field Content Alert Condition",
                 "https://www.graylog.org/",
-                "This condition is triggered when the aggregated value of a field is higher/lower than a defined "
-                + "threshold for a given time range."
+                "This condition is triggered when the content of messages is equal to a defined value."
```

Comment:
Ok, then let's leave them in and figure out what to do with it at a later stage.